scout/server/blueprints/cases/controllers.py

```diff
@@ -855,15 +885,15 @@ def mt_excel_files(store, case_obj, temp_excel_dir):
             for col, field in enumerate(line):  # each field in line becomes a cell
                 Report_Sheet.write(row, col, field)
 
-        # coverage_stats is None if app is not connected to Chanjo or {} if samples are not in Chanjo db
-        if coverage_stats and sample_id in coverage_stats:
-            # Write coverage stats header after introducing 2 empty lines
-            for col, field in enumerate(MT_COV_STATS_HEADER):
-                Report_Sheet.write(row + 3, col, field)
+        row += 3
 
-            # Write sample MT vs autosome coverage stats to excel sheet
-            for col, item in enumerate(["mt_coverage", "autosome_cov", "mt_copy_number"]):
-                Report_Sheet.write(row + 4, col, coverage_stats[sample_id].get(item))
+        row = write_coverage(
+            Report_Sheet, row, "Coverage Stats (chanjo2)", chanjo2_coverage_stats, sample_id
+        )
+
+        row = write_coverage(
```

Comment:
Chanjo stats look exactly like the chanjo2 stats, or, more correctly, the other way around.